### DISCUSSION TITLE
Getting plugins into your repository without PR

### DISCUSSION THREAD

**Discussion by zuckung** [Hecter94/EndlessSky-PluginArchive#69] [General]
I would love to put some of my own small plugins into your repository. Unfortunately i m suffering from unfortunate circumstances.

Currently i have no pc available and testing around on an old tablet. Allthough i have a big sdcard, i can't get a git client to clone your repository on it to make PRs, and the internal space is too low. The original commandline git, which should accept my sdcard, refuses to work. The web editor at github.dev bugs around on my tablet. And editing 1 file after another is a pain, and creats massive commits. Not really sure if a mass of commits is a problem, like said before i m pretty new to github, still trying to understand how it works. 

Anyway, are you open for direct zip delivery of files, which you have to put in by your own? Or maybe a single zip with plugins allready in the right folder structure?

To contribute to your work, i can offer to work on the pluginmanifest. Like putting in descriptions and links. Single file edit is acceptable for my tablet enviroment, even if it take some hours.

**Comment by Hecter94** [comment]
If you send me the zip I can upload it manually. 
If the zip's too large for GitHub to handle, I'm also on the Endless Sky discord which can handle up to 50mb. https://discord.gg/ZeuASSx
